app/forms/openid_connect_authorize_form.rb

```diff
@@ -100,6 +100,8 @@ def link_identity_to_service_provider(current_user, rails_session_id)
       rails_session_id: rails_session_id,
       ial: ial_context.ial,
       aal: aal,
+      acr_values: acr_values,
```

Comment:
I think that here we want to `acr_values.join(' ')` to ensure they are able to be properly parsed by the `Vot::Parser`. It expects these to be separated by spaces.